grow-135 assests, structure, styles.

BoulderBrains opened the pull request:
Got most of this in lender quotes section for the lend by category homepage built today. 
It currently looks like this: 
**Mobile**
![Screen Shot 2020-07-24 at 5 15 02 PM](https://user-images.githubusercontent.com/1521381/88444270-40f6da80-cdd1-11ea-9b81-fa301911deca.png)

**Desktop**
![Screen Shot 2020-07-24 at 5 14 46 PM](https://user-images.githubusercontent.com/1521381/88444268-3f2d1700-cdd1-11ea-9f67-3d53339f3978.png)

The main piece left is getting the card background image into place. 

Just a draft PR currently feel free to make any comments you've got.

ryan-ludwig commented:
We can improve the file size of the flourish assets. They should end up under 10k for sure. With the weathered texture look they have it makes SVGs really huge. I had good luck exporting them in Photoshop with PNG-8 at 16 colors and then running those PNGs through [ImageOptim](https://imageoptim.com/).

ryan-ludwig commented on src/pages/Homepage/LendByCategoryHomepage.vue:
```diff
@@ -157,6 +157,47 @@
 					What our lending <br class="so mo"> community thinks
 				</h2>
+				<div
+					class="lender-quotes__lender-quote--card small-10 medium-8 large-5 xlarge-4 columns"
```
I would recommend making the cards themselves their own BEM "block" since they could stand alone easily without the lender-quotes section.. Maybe .quote-card or something.

ryan-ludwig commented on src/pages/Homepage/LendByCategoryHomepage.vue:
```diff
@@ -157,6 +157,47 @@
+					>
+					</div>
+					<!-- TODO:
+					This is a little tricky, need to put more thought in here
```
Oh interesting, you could add a modifier class with the loop index on the quote-card. And then in CSS use the index to position them.

Sortof like 
```
<div :class="quote-card quote-card--{index}">
  <div :class="quote-card__flourish">
  ...

.quote-card__flourish {
  position: absolute;
}

.quote-card--1 .quote-card__flourish {
  top: 1rem;
  left: 1rem;
  width: 3rem;
}
.quote-card--2 quote-card__flourish {
  top: 2rem;
  right: 0;
  width: 2.5rem;
}
```

eddieferrer replied:
☝️ Like @ryan-ludwig  said. You could also target with `:nth-child(odd)` `:nth-child(even)` if you wanted to support an arbitrary number of quotes,

eddieferrer commented on src/pages/Homepage/LendByCategoryHomepage.vue:
```diff
@@ -157,6 +157,47 @@
+						loading="lazy"
+						alt=""
+					/>
+					<div
```
I'm not sure that these should be divs. To keep it semantic, I think they should be a combination of p's and spans. Is there any reason to use v-html here instead of just template interpolation {{ }} ?

BoulderBrains replied:
Nope no specific reason Eddie, thanks for bringing this up, I've made that change after reading a little on this. (https://forum.vuejs.org/t/performance-v-html-vs-interpolation-aka-braces/29559/3)

eddieferrer replied:
@BoulderBrains 👍  theres also some XSS security issues with v-html, they wouldnt apply in this case, but something to be aware of: https://vuejs.org/v2/guide/syntax.html#Raw-HTML

BoulderBrains commented:
Adjustments mentioned above have been addressed. 

:
![Screen Shot 2020-07-27 at 4 38 43 PM](https://user-images.githubusercontent.com/1521381/88602447-a80dcc80-d027-11ea-9273-101e2753fec8.png)

:
![Screen Shot 2020-07-27 at 4 38 01 PM](https://user-images.githubusercontent.com/1521381/88602451-a8a66300-d027-11ea-9787-674253f9d2be.png)

ryan-ludwig commented on src/pages/Homepage/LendByCategoryHomepage.vue:
```diff
@@ -459,6 +506,80 @@ export default {
+	}
+
+	// Couldn't get BEM working in here
+	.quote-card:nth-child(even) {
```
Yeah, this is a sass thing I'm afraid. What you've got is good.

In sass the` &` really just means append that to the parent selector in SASS. It makes it easy to write BEM, but it's not really BEM specific.
```
.hi {
  &-there {
```

would output `.hi-there` 

```
.hi:nth-child(even) {
  &-there {
```
would try to output `.hi:nth-child(even)-there` but fail because it's an invalid selector.


So in your case you could do
```
.quote-card:nth-child(even) {
  .quote-card {
    &__flourish { ...
```

But that seems excessive. What you've got is readable to me.

BoulderBrains replied:
Thanks for this explanation @ryan-ludwig

ryan-ludwig commented on src/pages/Homepage/LendByCategoryHomepage.vue:
```diff
@@ -157,6 +157,33 @@
+					/>
+					<img
+						:src="lenderQuote.image"
+						class="quote-card__lender-img"
```
adding `loading="lazy"` and `:alt="lenderQuote.attribution"` would be good here.

ryan-ludwig commented:
@BoulderBrains I think adding a `z-index: -1;` to all the flourishes will safeguard overlaps.

eddieferrer commented:
@BoulderBrains yellow flourish still visible under card at ~999px. I think you can add a background of white to `.quote-card`  to take care of the issue

BoulderBrains commented:
Thanks for the review @eddieferrer, I believe that all of your raised issues have been addressed in the last few commits. 
- Took out the xlarge-4 class, so quote-cards remain at the large-5 size all the way up. 
- Fixed the flourish overlap issue, good catch on that one, thank you. 
**Questions:**
- I fixed the margin between the two card by manually setting the margin-right and margin-left on the odd/even card definitions: 
```.quote-card:nth-child(even) {
		@include breakpoint(large) {
			margin-right: 0.9rem;  <--- HERE
		}

		.quote-card__flourish {
			top: 0;
			left: 0;
			width: 10rem;

			@include breakpoint(large) {
				width: 8rem;
			}
		}
	}

	.quote-card:nth-child(odd) {
		@include breakpoint(large) {
			margin-left: 0.9rem; <--- HERE
		}

		.quote-card__flourish {
			bottom: 4rem;
			right: 0;
			width: 7.5rem;

			@include breakpoint(large) {
				width: 5.5rem;
			}
		}
	}```
Would it be prefered to try and mess around with the offset and incomplete row options in foundation? 

- Changing the width of the cards makes the left card and the .lender-quotes__flourish to overlap. I should be able to fix this with z-index as both the .quote-card(position: relative) and .lender-quotes__flourish(position: absolute) are position with a definition that meets the z-index requirements, however this doesn't seem to be working. Seeking advice there. 

Let me know what you think, thanks again for the review.

ryan-ludwig commented on src/pages/Homepage/LendByCategoryHomepage.vue:
```diff
@@ -439,7 +487,9 @@ export default {
 	&__header {
+		margin-bottom: 2rem;
 		font-weight: bold;
+		z-index: 1;
```
not needed

ryan-ludwig commented on src/pages/Homepage/LendByCategoryHomepage.vue:
```diff
@@ -459,6 +509,89 @@ export default {
 			pointer-events: none;
```
needs a z-index: -1

ryan-ludwig commented on src/pages/Homepage/LendByCategoryHomepage.vue:
```diff
@@ -459,6 +509,89 @@ export default {
+			line-height: 1.3;
+			font-weight: bold;
+			margin-bottom: 0;
+			z-index: 100;
```
not needed

ryan-ludwig commented on src/pages/Homepage/LendByCategoryHomepage.vue:
```diff
@@ -459,6 +509,89 @@ export default {
+		}
+
+		&__title {
+			z-index: 100;
```
not needed

BoulderBrains commented:
![Screen Shot 2020-07-28 at 2 06 17 PM](https://user-images.githubusercontent.com/1521381/88721649-8a9a3a80-d0db-11ea-983e-9f89a0dbdd5c.png)

![Screen Shot 2020-07-28 at 2 05 48 PM](https://user-images.githubusercontent.com/1521381/88721665-8c63fe00-d0db-11ea-8e77-b68513e174bd.png)

BoulderBrains commented:
Thank you both 🙏
